docs/semgrep-secrets/getting-started.md

```diff
@@ -66,11 +66,39 @@ The **Secrets** page consists of:
   - Whether it is a [historical finding](/semgrep-secrets/historical-scanning)
   - For users of the Semgrep Jira integration: whether there is a Jira ticket that accompanies that finding
 
+### Group findings
+
+By default, Semgrep groups all of the findings by the rule Semgrep used to match the code. This view is called the **Group by rule** view.
+
+Semgrep sorts findings by severity. For a given severity, Semgrep further sorts findings as follows:
+
+1. Findings generated by custom rules
+2. Issue count in descending order
+3. Findings ID in ascending order
+
+![Findings grouped by rule](/img/secrets-findings.png#md-width)
+
+To view findings individually, toggle **Group by Rule** to **No grouping** using the drop-down menu in the header. Findings are displayed based on the date they were found, with the most recent finding listed at the top.
+
+![Group by Rule option](/img/cloud-platform-findings-no-grouping.png#md-width)
+
 ### Filter findings
 
-Semgrep AppSec Platform offers a variety of filters to help you narrow down the
-list of findings. The following sections describe what filters are available to
-you.
+Use filters to narrow down your results. The following criteria are available for filtering:
+
+
+| Filter                    | Description  |
+| ------------------------  | ------------ |
+| **Projects**              | Filter by repositories connected to Semgrep AppSec Platform. |
+| **Branches**              | Filter by findings in different Git branches. |
+| **Teams**                 | Filter for findings in projects to which the specified teams are associated with. Available only to organizations with RBAC enabled. |
```

Comment:
```suggestion
| **Teams**                 | Filter for findings in projects to which the specified teams are associated with. Available only to organizations with [Semgrep Teams](/deployment/teams#teams-beta) enabled. |
```

We have 2 kinds of RBAC:
- One which is enabled by default and provides 2 roles (admin/member)
- One which is manually enabled by the user and provides 3 roles (admin/member/manager) **and** the concept of teams